ubo_app/services/090-web-ui/web-app/src/inputs.tsx

```diff
@@ -56,45 +56,50 @@ export function Inputs({
     )?.value;
 
     if (action === "provide") {
-      const inputResult = new InputResult();
-      inputResult.setMethod(InputMethod.INPUT_METHOD_WEB_DASHBOARD);
+        const inputResult = new InputResult();
```

Comment:
I think the text editor has accidentally changed the indentation here after saving the file.
Some lines, randomly, have new indentation, please revert it.